playbook/app/pb_kits/playbook/pb_avatar/_avatar.tsx

```diff
@@ -7,7 +7,7 @@ import { globalProps, GlobalProps } from '../utilities/globalProps'
 import Image from '../pb_image/_image'
 import OnlineStatus from '../pb_online_status/_online_status'
 
-type AvatarProps = {
+export type AvatarProps = {
```

Comment:
This is awesome! So much cleaner than repeating the same types across multiple kits